client/src/pages/HomePage/HomePage.js

```diff
@@ -2,24 +2,31 @@ import React, { useEffect, useState } from 'react';
 import styles from './Home.module.css';
 import RecipePreview from '../../components/RecipePreview/RecipePreview';
 import axiosInstance from '../../services/api'
+import { useAuthContext } from '../../hooks/useAuthContext';
 
 const Home = () => {
   const [recipes, setRecipes] = useState([]);
   const [error, setError] = useState(null);
+  const { user } = useAuthContext();
 
   useEffect(() => {
     // Fetch recipes when the component mounts
     const fetchRecipes = async () => {
       try {
-        const response = await axiosInstance.get('/recipes');
+        const response = await axiosInstance.get('/recipes',{headers:
+          {Authorization: `Bearer ${user.token}`}
+        });
         setRecipes(response.data.recipes);
       } catch (err) {
         setError('Failed to fetch recipes');
         console.error(err);
       }
     };
-    fetchRecipes();
-  }, []);
+    if(user){
```

Comment:
Here we said that we're gonna display the recipes no matter if there is a user logged in or not, no?